contents/handbook/why-does-posthog-exist.md

```diff
@@ -34,35 +29,33 @@ Third, we’ve managed this very well so far.
 
 ## Our strategy
 
-### 1. Be the source of truth for customer and product data
+### 1. Be the source of truth for all product context
 
-Building a successful product is hard; doing so when you don't understand your customers is even harder. It's wild that no one has already provided a complete customer record. This has happened because the entire industry has focused on integration instead of consolidation.
+Building a successful product is hard; doing so when you don't understand your customers is even harder. It's wild that no one has already provided a complete record of everything engineers need to ship products. This has happened because the entire industry has focused on integration instead of consolidation.
 
-Traditionally, as companies scale, their data warehouse becomes the source of truth, and non-warehouse native tools (like product analytics) become less relevant as people lose trust in the data they collect, simply because they are misused and divorced from the source of truth. Every company winds up with a huge mess of data spaghetti, with their business logic still spread across dozens or hundreds of tools.
+Traditionally, as companies scale, their data warehouse becomes the source of truth, and non-warehouse native tools (like product analytics) become less relevant as engineers lose trust in the data they collect, simply because they are misused and divorced from the source of truth. Every company winds up with a huge mess of data spaghetti, with their business logic still spread across dozens or hundreds of tools.
```

Comment:
We de-emphasized the importance of this in our chat, but I still think it's a really important point, so I'm glad you left it in. I think we should still aim for this.